### ISSUE TITLE
Duplicate unique ids in ON01

### ISSUE THREAD

**Issue by edwardsmarc** [edwardsmarc/CASFRI#644]
Many polyid are null in on01

6425 polyids contain {1} which causes errors. Remove these characters during loading.

There are also 6 cases of duplicated ids. Both recno and polyid are duplicated in these cases:

```
with a as(select concat(recno, src_filename)_id from rawfri.on01 group by (recno, src_filename) HAVING count(*)>1)
select * from rawfri.on01, a where concat(recno, src_filename) IN(a._id)
```

**Comment by edwardsmarc** [comment]
@pedrogit 
I fixed the {1} error above. There are still many polyids that become NULL during loading with your script. 

They don't effect the cas_id but polyid is used as the original source id in the cas table.

**Comment by edwardsmarc** [comment]
Update: the issue is around the ODBC driver. ~400000 polyids are set to null on loading. I've dropped polyid and instead use recno and src_filename in the cas_id, and recno as the orig_stand_id.

This results in 6 duplicate cas_ids.

**Comment by pedrogit** [comment]
Fixed in 5.3.0. Closing
